service/src/main/java/uk/gov/hmcts/reform/fpl/service/ValidateSupportingEvidenceBundleService.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.validation.groups.DateOfIssueGroup;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ValidateSupportingEvidenceBundleService {
+    private final ValidateGroupService validatorService;
+
+    public <T> List<String> validateBundle(List<T> documentBundle) {
+        return documentBundle.stream()
+            .map(e -> validatorService.validateGroup(e, DateOfIssueGroup.class))
```

Comment:
I am not sure if reusing some group is good thing here, are we sure it's the same meaning as that group should denote? Alternatively I am not sure if you need even group at all - you are just validating single class, do you need to pass any group? 

Seems a bit confusing to me!

Reply:
not needed, you are right. Removed.